### PR TITLE
Update CVE-2020-10264.json

### PR DESCRIPTION
**Pull request by vmayoral**
From https://github.com/aliasrobotics/RVD/issues/1444.

### CODE REVIEW

**Comment by cve-team** [comment]
vmayoral is not recognized currently as an account participating in the Git Pilot. To ensure timely processing of your pull request, please contact the CVE Team using either the CVE Request web form (https://cveform.mitre.org/) or an email (cve@mitre.org) and confirm your identity.

File: 2020/10xxx/CVE-2020-10264.json [hunk 1]
```diff
@@ -3,16 +3,107 @@
     "data_format": "MITRE",
```
Comment (cve-team) [inline 2020/10xxx/CVE-2020-10264.json:3]: MISC or CONFIRM, but name != url for reference.

Comment (vmayoral) [reply 2020/10xxx/CVE-2020-10264.json]: Reviewed. Updated.

File: 2020/10xxx/CVE-2020-10264.json [hunk 2]
```diff
@@ -1,18 +1,106 @@
 {
```
Comment (cve-team) [inline 2020/10xxx/CVE-2020-10264.json:1]: MISC or CONFIRM, but name != url for reference.

**Comment by vmayoral** [comment]
I don't quite understand where's the problem. The `cmdlinejsonvalidator.py` script seems to be happy with it:

```bash
$ wget https://raw.githubusercontent.com/CVEProject/cvelist/578a20ea57748aeb936566edcdf0888efbae71ab/2020/10xxx/CVE-2020-10264.json
<output ommitted>
$ python3 cmdlinejsonvalidator.py CVE-2020-10264.json ../cve_json_schema/CVE_JSON_4.0_min_public.schema
Record passed validation
```

Some help would be appreciated.

**Comment by cristina479** [comment]
That's strange. Please change the reference_data to the following and the error will go away:

"references": {
        "reference_data": [
            {
                "name": "https://www.universal-robots.com/how-tos-and-faqs/how-to/ur-how-tos/real-time-data-exchange-rtde-guide/",
                "refsource": "CONFIRM",
                "url": "https://www.universal-robots.com/how-tos-and-faqs/how-to/ur-how-tos/real-time-data-exchange-rtde-guide/"
            }
        ]
    }

File: 2020/10xxx/CVE-2020-10264.json (hunk 2 above)
Comment (cve-team) [inline 2020/10xxx/CVE-2020-10264.json:1]: MISC or CONFIRM, but name != url for reference.

**Comment by vmayoral** [comment]
@cristina479 many thanks for the help. Tried following your indications at https://github.com/CVEProject/cvelist/pull/3514/commits/0b7541e1a3eb00cf609af06aebc1dacb920a0ba1 but it seems I failed. Mind giving it a look again? How can I reproduced the automatic reviews locally to avoid troubling you any further?

t

**Comment by cristina479** [comment]
> @cristina479 many thanks for the help. Tried following your indications at [0b7541e](https://github.com/CVEProject/cvelist/commit/0b7541e1a3eb00cf609af06aebc1dacb920a0ba1) but it seems I failed. Mind giving it a look again? How can I reproduced the automatic reviews locally to avoid troubling you any further?

This is probably an internal check outside the schema, so you won't be able to reproduce it locally. Could you apply the change I suggested, commit it, and push it?

**Comment by cristina479** [comment]
> Sure, @cristina479 see [vmayoral@5880d9c](https://github.com/vmayoral/cvelist/commit/5880d9c04e3265a59901e800e70c6abf8f88a9fe). I fixed as you indicated above (though I’d prefer the name as per my previous commit), am I still missing something and if so, what exactly? Thanks for your patience.
> 
> I’m trying to set up a pseudo-automatic system to report tickets however seeing how much I’m struggling with the manual process, I’m growing scared.

Your changes fixed the error. Before merging the PR I have a quick question, would it be more accurate to say that CB3 SW Versions 3.3 up to 3.12.1 and e-series SW Versions 5.0 up to 5.7 are the affected versions?

**Comment by vmayoral** [comment]
> Your changes fixed the error. Before merging the PR I have a quick question, would it be more accurate to say that CB3 SW Versions 3.3 up to 3.12.1 and e-series SW Versions 5.0 up to 5.7 are the affected versions?

Thanks @cristina479, adapted accordingly

**Comment by vmayoral** [comment]
Sorry for that last bit, pushed to the wrong branch. Fixed now. All yours @cristina479

**Pull request merged**